<ui version="4.0" >
 <class>form_smbstatusl</class>
 <widget class="QMainWindow" name="form_smbstatusl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>qtsmbstatus</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../client/client.qrc" >
    <normaloff>:/icons/samba.png</normaloff>:/icons/samba.png</iconset>
  </property>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>31</y>
     <width>330</width>
     <height>577</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QTreeWidget" name="treeWidget" >
      <column>
       <property name="text" >
        <string/>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="Configure" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="action_View_log" />
    <addaction name="configureConfigure_clientAction" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="Search" >
    <property name="title" >
     <string>&amp;Search</string>
    </property>
    <addaction name="searchAll" />
    <addaction name="separator" />
    <addaction name="searchNextAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpAboutAction" />
    <addaction name="helpAbout_QtAction" />
   </widget>
   <addaction name="Configure" />
   <addaction name="Search" />
   <addaction name="helpMenu" />
  </widget>
  <action name="helpAboutAction" >
   <property name="icon" >
    <iconset resource="../client/client.qrc" >
     <normaloff>:/icons/samba.png</normaloff>:/icons/samba.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;About QtSmbstatus</string>
   </property>
   <property name="iconText" >
    <string>&amp;About QtSmbstatus</string>
   </property>
   <property name="toolTip" >
    <string>&amp;About QtSmbstatus</string>
   </property>
   <property name="statusTip" >
    <string>About QtSmbstatus</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAbout_QtAction" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
   <property name="iconText" >
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip" >
    <string>About &amp;Qt</string>
   </property>
   <property name="statusTip" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="configureConfigure_clientAction" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../client/client.qrc" >
     <normaloff>:/icons/configure.png</normaloff>:/icons/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Configure client</string>
   </property>
   <property name="iconText" >
    <string>&amp;Configure client</string>
   </property>
   <property name="statusTip" >
    <string>Configure QtSmbstatus client</string>
   </property>
  </action>
  <action name="searchMachineAction" >
   <property name="text" >
    <string>&amp;Machine</string>
   </property>
   <property name="iconText" >
    <string>&amp;Machine</string>
   </property>
   <property name="statusTip" >
    <string>Search machine</string>
   </property>
  </action>
  <action name="searchUserAction" >
   <property name="text" >
    <string>&amp;User</string>
   </property>
   <property name="iconText" >
    <string>&amp;User</string>
   </property>
   <property name="statusTip" >
    <string>Search user</string>
   </property>
  </action>
  <action name="searchShare_openAction" >
   <property name="text" >
    <string>Share &amp;open</string>
   </property>
   <property name="iconText" >
    <string>Share &amp;open</string>
   </property>
   <property name="statusTip" >
    <string>Search share open</string>
   </property>
  </action>
  <action name="searchFile_openAction" >
   <property name="text" >
    <string>&amp;Locked file</string>
   </property>
   <property name="iconText" >
    <string>&amp;Locked file</string>
   </property>
   <property name="statusTip" >
    <string>Search file open</string>
   </property>
  </action>
  <action name="searchNextAction" >
   <property name="text" >
    <string>&amp;Next</string>
   </property>
   <property name="iconText" >
    <string>&amp;Next</string>
   </property>
   <property name="statusTip" >
    <string>Next</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="searchAll" >
   <property name="text" >
    <string>&amp;Find text</string>
   </property>
   <property name="iconText" >
    <string>&amp;Find text</string>
   </property>
   <property name="statusTip" >
    <string>Find text</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="icon" >
    <iconset resource="../client/client.qrc" >
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit QtSmbstatus</string>
   </property>
  </action>
  <action name="action_View_log" >
   <property name="text" >
    <string>&amp;Show CIFS/SMB activities</string>
   </property>
  </action>
  <action name="action_Connect" >
   <property name="icon" >
    <iconset resource="../client/client.qrc" >
     <normaloff>:/icons/connect_no.png</normaloff>:/icons/connect_no.png</iconset>
   </property>
   <property name="text" >
    <string>Co&amp;nnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../client/client.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>helpAboutAction</sender>
   <signal>activated()</signal>
   <receiver>form_smbstatusl</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpAbout_QtAction</sender>
   <signal>activated()</signal>
   <receiver>form_smbstatusl</receiver>
   <slot>helpAboutQt()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configureConfigure_clientAction</sender>
   <signal>activated()</signal>
   <receiver>form_smbstatusl</receiver>
   <slot>ConfigureSlot()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchNextAction</sender>
   <signal>activated()</signal>
   <receiver>form_smbstatusl</receiver>
   <slot>NextSlot()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchAll</sender>
   <signal>activated()</signal>
   <receiver>form_smbstatusl</receiver>
   <slot>AllSearchSlot()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
